examples/ThemeMaker.py

```diff
@@ -0,0 +1,271 @@
+import tkinter
+import customtkinter
+from tkinter.colorchooser import askcolor
+from tkinter import filedialog, messagebox
+import json
+
+#Main Window
+WIDTH = 390
+HEIGHT = 350
+customtkinter.set_appearance_mode("Dark")
+customtkinter.set_default_color_theme("blue")
+self=customtkinter.CTk()
+self.title("CustomTkinter ThemeMaker")
+self.geometry(f"{WIDTH}x{HEIGHT}")
+self.resizable(width=False, height=False)
+
+null=None #Default is null

```

Comment:
There's no need to define a null variable, just use None inside the dict definition or call the variable default_value or something

Reply:
Done